### PR TITLE
Terminology Updates

### PR DESCRIPTION
**Pull request by cwarnermm**
#### Summary
Following the v6.0 release, where Account Settings options were split between 2 dialogs: Account Settings and Settings, the v6.2 release will further adjust documentation terminology to make a clear distinction between Profile & Security settings versus Channels-specific settings.

Updated references to Account Settings and Main Menu to match the latest product interface, and updated links to product documentation pages.

#### Ticket Link
https://community-daily.mattermost.com/boards/workspace/g4k9s1f953n4bytz5f9sz9orer/bm1j7wcrrspoymkjermbydid97h/vcwyy7xnhh4qits4thrb3pte53e/7e84xw9oxy9ozca7g7ejyhyacmy

#### Related Tickets
https://github.com/mattermost/docs/pull/5286

### CODE REVIEW

File: site/content/contribute/webapp/e2e-cheatsheet/main-menu.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
 < <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type">string</a> >
-  - Possible values: `'Account Settings'`, `'Invite People'`, `'Team Settings'`, `'Manage Members'`, `'Create a Team'`, `'Join Another Team'`, `'Leave Team'`, `'Integrations'`, `'Plugin Marketplace'`, `'System Console'`, `'Help'`, `'Keyboard Shortcuts'`, `'Report a Problem'`, `'Download Apps'`, `'About Mattermost'` and `'Logout'`.
+  - Possible values: `'Channels'`, `'Boards'`, `'Playbooks'`, `'System Console'`, `'Integrations'`, `'Marketplace'`, `'Download Apps'`, and `'About Mattermost'`.
 
-##### Open the Main Menu
+##### Open the Product menu
 ```javascript
-// # Open 'Main Menu'
+// # Open 'Product menu'
 cy.uiOpenMainMenu();
```
Comment (cwarnermm) [inline site/content/contribute/webapp/e2e-cheatsheet/main-menu.md:22]: @saturninoabril - Given the terminology change from "Main Menu" to "Product menu" -- can you assist with updates on this page, please? I've updated all text-based references. Can you update the code-specific references?

**Comment by hanzei** [comment]
@cwarnermm I'm not sure I'm the right one to review this. Could you please ask someone with more knowledge about the product UI?

**Comment by cwarnermm** [comment]
Absolutely, @hanzei. Thanks!

File: site/content/contribute/webapp/e2e-cheatsheet/main-menu.md (hunk 1 above)
Comment (saturninoabril) [reply site/content/contribute/webapp/e2e-cheatsheet/main-menu.md]: Sure @cwarnermm! Please see changes made at https://github.com/mattermost/mattermost-developer-documentation/pull/972/commits/52a50fb04f97d0874d14833a3f0980c93a0fa8a3

Comment (cwarnermm) [reply site/content/contribute/webapp/e2e-cheatsheet/main-menu.md]: Thank you, @saturninoabril!

**Review by saturninoabril — APPROVED** [review]
LGTM, thanks Carrie!

File: site/content/integrate/admin-guide/admin-personal-access-token/_index.md [hunk 2]
```diff
@@ -35,25 +35,25 @@ Then select **Save**.
 
 ## Revoking a Personal Access Token
 
-A personal access token can be revoked by deleting the token from either the user's account settings or from the System Console. Once deleted, all sessions using the token are deleted, and any attempts to use the token to interact with the Mattermost server are blocked.
+A personal access token can be revoked by deleting the token from either the user's profile or from the System Console. Once deleted, all sessions using the token are deleted, and any attempts to use the token to interact with the Mattermost server are blocked.
```
Comment (hmhealey) [inline site/content/integrate/admin-guide/admin-personal-access-token/_index.md:38]: Shouldn't this be "profile settings" and not just "profile"? I feel like "user's profile" has a different meaning than the profile settings

File: site/content/integrate/admin-guide/admin-zapier-integration/_index.md [hunk 3]
```diff
@@ -58,28 +58,28 @@ You'll be provided with a **Client ID** and **Client Secret**. Save these values
...
    - **Connect the Action Account:** Select **Connect a New Account** and enter the following fields:
      - **Mattermost URL:** This is the URL you use to access your Mattermost site. Don't include a slash at the end of the URL and don't append a team to the end of the server URL. For example, `https://community.mattermost.com/core` is the entire URL to the Contributors team on our community server. The **Mattermost URL** entered here would be `https://community.mattermost.com`.
-     -  **Client ID/Secret:** If Zapier has been enabled as an OAuth application as per the steps above, then these values can be found by navigating to one of your Mattermost teams, then **Main Menu > Integrations > OAuth 2.0 Applications**. Select **Show Secret** next to the Zapier app, then obtain the Client ID and Client Secret.
+     -  **Client ID/Secret:** If Zapier has been enabled as an OAuth application as per the steps above, then these values can be found by navigating to one of your Mattermost teams, then **Product menu > Integrations > OAuth 2.0 Applications**. Select **Show Secret** next to the Zapier app, then obtain the Client ID and Client Secret.
```
Comment (hmhealey) [inline site/content/integrate/admin-guide/admin-zapier-integration/_index.md:61]: Is the correct capitalization here "Product menu" or "Product Menu"? We used to capitalize both words, and I would've expected that we'd continue doing that now

File: site/content/integrate/admin-guide/admin-personal-access-token/_index.md (hunk 2 above)
Comment (cwarnermm) [reply site/content/integrate/admin-guide/admin-personal-access-token/_index.md:38]: Great catch! Thank you!

Comment (cwarnermm) [inline site/content/integrate/admin-guide/admin-personal-access-token/_index.md:38]:
```suggestion
A personal access token can be revoked by deleting the token from either the user's profile settings or from the System Console. Once deleted, all sessions using the token are deleted, and any attempts to use the token to interact with the Mattermost server are blocked.
```

File: site/content/integrate/admin-guide/admin-zapier-integration/_index.md (hunk 3 above)
Comment (cwarnermm) [reply site/content/integrate/admin-guide/admin-zapier-integration/_index.md:61]: Good question. Yes, historically UI components were capitalized along with the component name. Going forward, we're no longer capitalizing UI components themselves (such as menu).

**Review by hmhealey — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**